Add the vcs option to the config file (#38)
yozhgoor/cargo-temp: README.md

    editor_args = [ "--wait", "--new-window" ]
    ```

### Use a VCS

> By default, cargo-temp will use Git as the default vcs for you projects, you can change that in the config file with the vcs option.

Reviewer: ```suggestion
By default, cargo-temp will use the default cargo VCS for your projects (which is normally git), you can change that in the config file with the `vcs` option.
```

```toml
vcs = "pijul"
```

The possible values are

> * Pijul
> * Fossil
> * Hg
> * None

Reviewer: It won't work with the capital cases

```suggestion
* pijul
* fossil
* hg
* none
* ... and all other option cargo normally propose
```

> If you do not add the option or if the value is `git` or something else that is not in the possible value, cargo-temp will use the default value.

Reviewer: ```suggestion
The `--vcs` value will be passed as is to cargo.
```
